avocado/utils/network/interfaces.py

```diff
@@ -571,3 +572,22 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def check_packet_loss(self, peer_ip, count, option=None, flood=False):
+        """
+        Function checks for packet loss during ping.
+        Return False for 0% packet loss and True if packet loss occurs.
```

Comment:
This docstring is not properly documented. @TasmiyaNalatwad can you please describe the arguments here?

Reply:
@beraldoleal  Thanks for the inputs. I will be adding the proper description for all the arguments used in the function.